test/e2e/.gitignore

```diff
@@ -19,3 +19,6 @@ results/
 
 # Cookie files
 cookies/
+
+/*.png
+/*.zip
```

Comment:
Hi, no, this only excludes images directly at the root and not in sub-folders. 

![image](https://user-images.githubusercontent.com/6851384/176511133-45ced9c4-0279-44ff-8ffc-ab472d38f420.png)

Reply:
Gotcha, thanks. I missed that it didn't have `**/` preceding it, oops.

Reply:
I'm assuming this `.gitignore` change fixes the issues with the e2e tests not passing? If so, please add a code comment and/or update the PR description so we document the reason for the change.

Reply:
@mpkelly, if this is an issue specific to running the tests, can we move the changes into a dedicated PR so we can clearly call out why we're making this change and note it's impact? This change is currently buried within this technically unrelated PR, and it will be difficult for future developers to work out why this change occurred.